test/e2e/playwright.config.ts

```diff
@@ -24,9 +24,9 @@ export default defineConfig( {
 	/* Fail the build on CI if you accidentally left test.only in the source code. */
 	forbidOnly: !! process.env.CI,
 	/* Retry on CI only */
-	retries: process.env.CI ? 1 : 0,
+	retries: process.env.CI ? 2 : 0,
 	/* Workers should use what is available */
-	workers: 1,
+	workers: process.env.CI ? 1 : '100%',
```

Comment:
I think you meant 50%?

Reply:
yes, I had it at 50% on this PR for first commit but wanted to make sure it wasn't making the other Playwright tests fail somehow, I have updated it back to 50% now